src/views/services/forms/DetailsTab.vue

```diff
@@ -88,6 +88,16 @@
           </template>
         </ck-image-input>
 
+        <ck-radio-input
+          :value="is_national"
+          @input="$emit('update:is_national', $event); $emit('clear', 'is_national')"
+          id="is_national"
+          :label="`National?`"
+          :hint="`Details if the service is only available at specific locations or can be accessed nationwide, e.g. via web or phone.`"
```

Comment:
No need to use interpolation here.

```suggestion
          label="National?"
          hint="Details if the service is only available at specific locations or can be accessed nationwide, e.g. via web or phone."
```

Reply:
Done.